custom_bash.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+cd /vdms/user_defined_operations/
+python3 -m pip install -r requirements.txt
+python3 -m pip install sk-video
+python3 -m pip install imutils
+python3 -m pip install zmq
```

Comment:
Let's add these dependencies to the `user_defined_operations/requirements.txt` if they are needed